lib/workers/global/index.ts

```diff
@@ -85,6 +85,7 @@ export async function start(): Promise<0 | 1> {
       await repositoryWorker.renovateRepository(repoConfig);
     }
     setMeta({});
+    globalCache.cleanup(config);
```

Comment:
Maybe we should call this in finally and use `redis.quit` instead of `end`

https://github.com/NodeRedis/node-redis/issues/714